docs/release_notes/5.0.28-24.md

```diff
@@ -0,0 +1,49 @@
+# Percona Server for MongoDB 5.0.28-24 ({{date.5_0_28}})
+
+[Installation](../install/index.md){.md-button}
+
+Percona Server for MongoDB 5.0.28-24 is an enhanced, source-available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for MongoDB 5.0.x Community Edition. 
+
+Percona Server for MongoDB 5.0.28-24 includes improvements and bug fixes of [MongoDB 5.0.28 Community Edition](https://www.mongodb.com/docs/manual/release-notes/5.0/#5.0.28---jul-15--2024) and supports its protocols and drivers.
+
+## Release Highlights
+
+This release of Percona Server for MongoDB includes the following features and improvements:
+
+### Enhanced Telemetry for better product usage reporting
+
+The enhanced telemetry feature provides comprehensive information about how it works, its components and metrics as well as updated methods how to disable telemetry. Read more in [Telemetry on Percona Server for MongoDB](../telemetry.md)
+
+### Reduce mean time to resolve (MTTR) compromised encryption key incidents in KMIP
+
+Starting with this release, Percona Server for MongoDB automatically activates the master encryption keys at startup and periodically checks (polls) their status in a KMIP server. If a master encryption key for a node is not in the Active state, the node reports an error and shuts down. This method allows security engineers to quickly identify which nodes require out-of-schedule master key rotation, such as in the case of compromised keys, without needing to rotate keys for the entire cluster. 
```

Comment:
This seems to be description from v8.0 but versions prior to v8.0 have different behavior. Let's check with @ktrushin

Reply:
The above text is almost correct if we are speaking about _new_ keys. I would add some clarifications:
- `activates the master encryption keys` -> ` activates all new master encryption keys`
- `is not in the Active state` -> `transitions from the Active to another state`
- add the following `If Percona Server for MongoDB detects at startup that data directory is encrypted with an existing key in the Pre-Active state, it logs a warning and continues as usual. An operations team is encouraged to transition the keys to Active state using master key rotation or manually.`